authorizer.py

```diff
@@ -133,19 +135,24 @@ def Redirect(self):
     for ss in serving_sessions:
       institution = ss.institution_name
       session = ss.session_name
-      login_type = ss.login_type
+      start_page = ss.start_page
       verified = self._VerifyStudent(institution,
                                      session,
                                      self.email)
       if verified:
```

Comment:
Here is a good place to check if they have access to multiple sessions (or institutions). We can send them to a new disambiguation page from here. There they can get a menu of all the institutions and sessions they have access to.